### PR TITLE
Added user stories for Nice-to-have and Unlikely-to-have

### PR DESCRIPTION
**Pull request by chuaweiwen**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |view my top five most accessed contact |find them more easily
+
+|`* *` |user |see frequently used contacts instantly |locate them more easily
+
+|`* *` |user |view my contact list online |so that I can view it without the device running the software
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: Should we clarify/specify how it is viewed? If we are not viewing it through the AddressBook application.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |send email to a group of contacts |easily send email to a group
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: "send email to a group of contacts with one command"
Would this be clearer?

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |include more details for a person's information |identify the person easily
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: "...for a person's contact information"
"store more information about a person and identify him/her more easily"

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |delete multiple users in one command |remove several entries that I no longer need quickly
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: "remove several contacts..."
I think we should try to be consistent in our usage of terms.

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |have color-coded commands and attributes in command line |identify the command and attributes easily
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: Attributes sounds a bit vague, but I can't find a suitable word LOL
Or do we want to use the term "input segment"?

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |view my recent additions |identify which contact is newly added
+
+|`* *` |lazy user |have short-hand form of command |perform commands quickly
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: "have short-hand form for each command"

File: docs/DeveloperGuide.adoc [hunk 7]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |set reminders |remind myself any activities or events that will happen
+
+|`* *` |user |save my contacts as an external file |backup or transfer data easily
+
+|`* *` |user |add multiple emails and phone numbers to a contact |handle contacts with more than one email and phone numbers
+
+|`* *` |user |add birthday of a person |store the birthday of the person
+
+|`* *` |user |add reminders of any approaching birthday |prepare for the person's birthday
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: I thought if we allow the adding of birthday information, the program would automatically be able to remind us of any impending birthdays? We need to explicitly add reminders for birthdays?

File: docs/DeveloperGuide.adoc [hunk 8]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |sort contact by birthday |identify the contacts who will have a birthday soon
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: "sort contact information by birthday"
"identify the persons who will..."

File: docs/DeveloperGuide.adoc [hunk 9]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: "user with the contact information of many persons..."
"sort contact information by name"

File: docs/DeveloperGuide.adoc [hunk 10]
```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`*` |sociable user |list all persons having a birthday of a particular month |identify all contacts who have the birthday in the same month
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc]: "...having birthday during a particular month"
"identify all persons who have birthday in the same month"

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (chuaweiwen) [reply docs/DeveloperGuide.adoc]: Good idea.

File: docs/DeveloperGuide.adoc [hunk 11]
```diff
@@ -731,7 +731,45 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *` |user |view my top five most accessed persons |find them more easily
```
Comment (tshradheya) [inline docs/DeveloperGuide.adoc]: Redundant with the next one. So delete this

File: docs/DeveloperGuide.adoc [hunk 12]
```diff
@@ -731,7 +731,45 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |lazy user |have short-hand form for each command |perform commands quickly
+
+|`* *` |user |set reminders |remind myself any activities or events that will happen
+
+|`* *` |user |save the Address Book as an external file |backup or transfer the Address Book easily
+
+|`* *` |user |add multiple emails and phone numbers to a person |handle persons with more than one email and phone numbers
+
+|`* *` |user |add reminders of any approaching birthday |prepare for the person's birthday
```
Comment (tshradheya) [inline docs/DeveloperGuide.adoc]: `get reminder` instead of add reminders

File: docs/DeveloperGuide.adoc [hunk 13]
```diff
@@ -731,7 +731,45 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |sort persons by birthday |identify the persons who will have a birthday soon
+
+|`*` |sociable user |list all persons having birthday during a particular month |identify all persons who have the birthday in the same month
```
Comment (tshradheya) [inline docs/DeveloperGuide.adoc]: Delete this. Too much focus on Birthday's

File: docs/DeveloperGuide.adoc [hunk 14]
```diff
@@ -731,7 +731,45 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`*` |user |count on the backup system |be assured that the Address Book can be recovered if lost
```
Comment (tshradheya) [inline docs/DeveloperGuide.adoc]: Redundant

File: docs/DeveloperGuide.adoc [hunk 15]
```diff
@@ -731,7 +731,45 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+|`*` |careless user |search a person using similar keywords |find contacts if I do not know the correct spelling
+
+|`*` |user |add same tag(s) to multiple persons in one command |tag multiple persons quickly
+
+|`*` |user |auto sort the list whenever I add a person |avoid using the sort command with every single additions
+
+|`*` |user |have a secured address book |only I have access to my Address Book
 |=======================================================================
 
 {More to be added}
```
Comment (tshradheya) [inline docs/DeveloperGuide.adoc]: Remove this

**Review by tshradheya — COMMENTED** [review]
Please have a look at comments.

**Pull request merged**